odoo/models.py

```diff
@@ -1851,7 +1851,7 @@ def _read_group_fill_temporal(self, data, groupby, aggregated_fields, annotated_
     def _read_group_prepare(self, orderby, aggregated_fields, annotated_groupbys, query):
         """
         Prepares the GROUP BY and ORDER BY terms for the read_group method. Adds the missing JOIN clause
-        to the query if order should be computed against m2o field. 
+        to the query if order should be computed against m2o field.
```

Comment:
It seems you have an automatic tool/linter that remove trailing spaces. While it is a good for new developments, we avoid changing it in existing files, it makes a bigger diff and creates potential conflicts.